### PR TITLE
artefact batch position update fix, update to int for translateX/Y an…

### PR DESCRIPTION
**Pull request by Bronson-Brown-deVost**
Here is the patch I promised. I had to update the database as well, since the problem had to do with us switching the translation x/y from uint to int. I updated the tests to check for this sort of error as well.

### CODE REVIEW

File: sqe-dto/Transformation.cs [hunk 1]
```diff
@@ -18,15 +18,15 @@ public class PlacementDTO
...
 
         [Required]
         public TranslateDTO translate { get; set; }
     }
 
     public class TranslateDTO
     {
-        public uint? x { get; set; }
-        public uint? y { get; set; }
+        public int? x { get; set; }
```
Comment (zmbq) [inline sqe-dto/Transformation.cs]: Why are these nullable if we have isPlaced? Under what circumstances are these going to be null?

**Review by zmbq — APPROVED** [review]
I'm going to approve this PR so that Shaindel can keep working on this, but I think translateX and Y should not be nullable in the DTO (and probably not in the database, either, but that's less my field here)

**Comment by zmbq** [comment]
I can't merge since the Tcoverage report is unavailable. So perhaps we should wait until we conclude the nullable X/Y issue, too.

**Comment by Bronson-Brown-deVost** [comment]
CI is giving us some trouble here too (hence no report to coveralls). I am getting a random error that I see others are also experiencing intermittently on TravisCI. I am just now fixing a deadlock issue that keeps popping up in the tests. We'll see if that helps.

**Comment by zmbq** [comment]
Perhaps this is, again, because of the memory demands of the backend and database, that Travis CI can't handle?

**Review by zmbq — APPROVED** [review]

**Comment by zmbq** [comment]
No coverage yet :-(

**Pull request merged**